posts/2023/03/16/kubernetes-meets-event-driven-ansible.md

```diff
@@ -0,0 +1,203 @@
+
+Event-Driven Ansible is designed using a concept called Rulebooks which consists of three main components:
+
+- Actions - Triggering the execution of assets including an Ansible Playbook or module 
+- Rules - Determination of whether received events match certain conditions 
+- Sources - Origination of events from external entities that are consumed and processed within the Ansible eventing framework
+
+There is a wide ecosystem of solutions available to manage Kubernetes from Ansible, which is provided primarily through the [kubernetes.core collection](https://github.com/ansible-collections/kubernetes.core). This Collection contains everything ranging from mechanisms to manage resources within a Kubernetes cluster, support for the Helm package manager to leveraging a Kubernetes cluster as an inventory source. There are capabilities made available now through the integration of Kubernetes and the Event-Driven Ansible framework. Event sources enable the consumption of changes originating from a Kubernetes cluster, which can be used to trigger automation to respond and act based on the received content and the configured rules. Let’s explore how to take advantage of this newly created capability to further the integration between Kubernetes and Ansible.
+
+The assets related to Event-Driven Ansible and Kubernetes are located within the [sabre1041.eda collection](https://galaxy.ansible.com/sabre1041/eda) within Ansible Galaxy. Ensure that the control node where the Ansible automation will be executed has the necessary tooling installed and configured. This includes Ansible Core, the tooling associated with Event-Driven Ansible, and the Collection containing the Event-Driven Ansible Kubernetes integration. Consult the associated documentation for both [Ansible Core](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) and [Event-Driven Ansible](https://www.ansible.com/blog/getting-started-with-event-driven-ansible) for the target Operating System and installation method.
```

Comment:
There is a link to the old blog which would break after the migration:
`[Event-Driven Ansible](https://www.ansible.com/blog/getting-started-with-event-driven-ansible)`

It can be replaced later, but mentioning it so we don't forget.